.github/dependabot.yml

```diff
@@ -142,6 +143,30 @@ updates:
       - dependency-name: org.eclipse.microprofile.openapi:microprofile-openapi-tck
       - dependency-name: org.eclipse.microprofile.opentracing:microprofile-opentracing-tck*
       - dependency-name: org.eclipse.microprofile.rest.client:microprofile-rest-client-tck
+      - dependency-name: org.eclipse.microprofile.reactive-streams-operators:microprofile-reactive-streams-operators-*
+      # Others
+      - dependency-name: com.puppycrawl.tools:checkstyle
+      - dependency-name: org.codehaus.plexus:plexus-archiver
```

Comment:
@aloubyansky ? There are also other Maven related dependencies that are not tracked (the resolver for instance IIRC). I'm not sure if we want to track them?

Reply:
Ideally, they should be compatible with the Maven version we depend on.

Reply:
OK, I'll remove this one then.